requirements/edx/github.txt

```diff
@@ -65,7 +65,6 @@ git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c
 # custom opaque-key implementations for CCX
 git+https://github.com/edx/ccx-keys.git@0.1.1#egg=ccx-keys==0.1.1
 
-git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
```

Comment:
We can delete this in a few months, when we are relatively certain no one else is using it. Or...can we redirect our fork back to the upstream repository.